modules/exploits/windows/local/ricoh_driver_privesc.rb

```diff
@@ -140,11 +138,6 @@ def exploit
       fail_with(Failure::BadConfig, 'The payload should use the same architecture as the target driver')
     end
 
-    @driver_path = ''
```

Comment:
I don't think this line should be deleted 👀

Reply:
Got that added back in 👍 , sorry about that.

Reply:
Oh closer inspection, it looks like the check method mutates this value - so deleting this assignment here works :+1:

I'm sure I knew this at the time 😄

Reply:
Got that taken back out 😄